integrated-scouting-apps/src/routes/pit.tsx

```diff
@@ -1,282 +1,128 @@
 import '../public/stylesheets/style.css';
 import logo from '../public/images/logo.png';
+import back from '../public/images/back.png';
 import { Checkbox, Flex, Form, Input, InputNumber, Select, Tabs, TabsProps, Upload, message } from 'antd';
 import { useEffect, useState } from 'react';
 import TextArea from 'antd/es/input/TextArea';
 import { RcFile, UploadChangeParam, UploadFile, UploadProps } from 'antd/es/upload/interface';
 import { LoadingOutlined, PlusOutlined } from '@ant-design/icons';
+import '../public/stylesheets/style.css';
+import no_image from '../public/images/no_image.png';
+import field_blue from '../public/images/field_blue.png';
+import field_red from '../public/images/field_red.png';
+import { useRef} from 'react';
+import {Button } from 'antd';
+import { ReactSketchCanvas, ReactSketchCanvasRef } from 'react-sketch-canvas';
+import { type } from '@testing-library/user-event/dist/type';
```

Comment:
whars this?

Reply:
i need to import button in order to use buttons